.travis.yml

```diff
@@ -20,15 +22,36 @@ matrix:
     - php: 7.3
     # 'php: nightly' is PHP 8.0
     - php: 7.4snapshot
+    - php: 7.3
+      env: SYMFONY="4.4.*"
+           ACTION="update"
+    - php: 7.3
+      env: SYMFONY="5.0.*"
+           ACTION="update"
   allow_failures:
     - php: 7.4snapshot
+    - php: 7.3
+      env: SYMFONY="4.4.*"
+           ACTION="update"
+    - php: 7.3
+      env: SYMFONY="5.0.*"
+           ACTION="update"
 
 before_install:
   - '[[ "$TRAVIS_PHP_VERSION" == "7.4snapshot" ]] || phpenv config-rm xdebug.ini'
   - composer self-update
+  # Set memory to max (memory fail)
+  - '[[ "$ACTION" == "install" ]] || echo "memory_limit=-1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini'
+  # Install before update because of memory usage
+  - '[[ "$ACTION" == "install" ]] || composer install'
+  # Set stability to dev to allow 4.4dev and 5.0dev
+  - '[[ "$ACTION" == "install" ]] || composer config minimum-stability dev'
+  # Choose version of symfony to use
+  - composer config extra.symfony.require $SYMFONY
```

Comment:
I suppose we want to execute it for not "install" as well, i.e. `[[ "$ACTION" == "install" ]] || composer config extra.symfony.require $SYMFONY `... and then probably we don't need global `SYMFONY="4.3.*"` at all for simplicity

Reply:
Yes, "composer config extra.symfony.require $SYMFONY" replace "4.3.*" by "4.3.*" for update  
It's just because :
- don't like  to use condition 
- prefere that all var are define for each case even if not use

Simplicity can be to have $ACTION and $SYMFONY always define. I had $STABILITY=dev (or stable) in the same spirit. I replace it by a condition because it works without it.

Reply:
Yeah, I understand it works, but it may just mislead other devs :/ My point is: if there's no effect - why execute those lines? Right now we would need to keep in sync both: `extra.symfony.require` in `composer.json` and global `$SYMFONY` in `.travis.yaml`, otherwise, it will put the repo in a dirty state on TravisCI. Yeah, probably still not a big deal, but why not simplify things and make it clearer for devs?

Reply:
Make sense : "Right now we would need to keep in sync both"
1) suppress global $SYMFONY
2) add condition on $ACTION 

So I have just to propose a new PR ?

Reply:
If we agree on it (probably good to wait for more opinions) - better to do changes in this PR  I think

Reply:
I don't have a strong opinion about this ... but regarding the mention to the "extra.symfony.require in composer.json", we can't remove that. It's a "best practice" of Symfony and want to promote it.

Reply:
extra.symfony.require in composer.json only change depending on symfony release to test. The discution was just to not replace 4.3 by 4.3 who has no utility.